docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -223,15 +225,15 @@ Sadly there is no option to do this via the AWS Console, so this step must be pe
 
 This will create a new certificate inside your AWS environment which can be reused for a variety of things.  Remember the name you chose for your certificate because we'll use that later in the setup process when we enable SSL.
 
-### Setup DNS CNAME (using AWS)
+### Setup DNS (using AWS)
 * Open up the AWS **Route 53** console by navigating to **Services > Networking > Route 53** in the AWS Console header
 * Click on **Hosted Zones** then click on the domain name you want to use for Metabase
 * Click on the blue button **Create Record Set** (a new panel will open up on the right side of the page)
 	* Enter in a **Name**: for your application.  this should be the exact url you plan to access Metabase with.  (e.g. metabase.mycompany.com)
-	* Under the dropdown for **Type**: select *CNAME - Canonical name*
-	* In the box labeled **Alias**: input the full path to your Elastic Beanstalk environment (e.g. mycompany-metabase.elasticbeanstalk.com)
+	* Under the dropdown for **Type**: select *A - IPv4 Address* (sometimes you can also choose CNAME - Canonical name but this may cause more trouble)
+	* In the box labeled **Alias**: choose your elastic bean environment in the drop-down or directly input the full path to your Elastic Beanstalk environment (e.g. mycompany-metabase.elasticbeanstalk.com)
```

Comment:
typo: `...elastic beanstalk environment...`

Reply:
should be fixed